### PR TITLE
EPMRPP-34251. Fix for edit launch permissions.

### PR DESCRIPTION
**Pull request by ArtsiomYeliseyenka**

### CODE REVIEW

File: app/src/controllers/user/selectors.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ export const userIdSelector = (state) => userInfoSelector(state).userId;
 export const settingsSelector = (state) => userSelector(state).settings;
 export const startTimeFormatSelector = (state) => settingsSelector(state).startTimeFormat;
 export const assignedProjectsSelector = (state) => userInfoSelector(state).assigned_projects || {};
+export const userAccountSelector = (state) => userInfoSelector(state).userRole || '';
```
Comment (DiscoElevator) [inline app/src/controllers/user/selectors.js]: userRoleSelector

File: app/src/common/utils/permissions/permissions.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ export const canDeleteUser = checkPermission(ACTIONS.DELETE_USER);
 export const canSeeMembers = checkPermission(ACTIONS.SEE_MEMBERS);
 export const canEditOwnAccount = checkPermission(ACTIONS.EDIT_OWN_ACCOUNT);
 export const canDeleteLaunch = checkPermission(ACTIONS.DELETE_LAUNCH);
+export const canEditLaunch = checkPermission(ACTIONS.EDIT_LAUNCH); // requires userRole, projectRole, isOwner.
```
Comment (DiscoElevator) [inline app/src/common/utils/permissions/permissions.js]: What is the purpose of this comment?

File: app/src/controllers/user/selectors.js (hunk 1 above)
Comment (ArtsiomYeliseyenka) [reply app/src/controllers/user/selectors.js]: misprint from my side. Better name is userAccountRoleSelector

File: app/src/common/utils/permissions/permissions.js (hunk 2 above)
Comment (ArtsiomYeliseyenka) [reply app/src/common/utils/permissions/permissions.js]: as i heard not all these check requires all 3 arguments. With a comment beside I'll know immediatly how many arguments I should pass for correct permission calculation.

**Pull request merged**